blog/2020-10-22-towards-debzium-ui.adoc

```diff
@@ -0,0 +1,88 @@
+= Towards a Graphical Debezium User Interface
+gmorling
+:awestruct-tags: [ community, discussion ]
+:awestruct-layout: blog-post
+
+Over the last five years, Debezium has become a leading open-source solution for change data capture for a variety of databases.
+Users from all kinds of industries work with Debezium for use cases like replication of data from operational databases into data warehouses, updating caches and search indexes, driving streaming queries via Kafka Streams or Apache Flink, synchronizing data between microservices, and many more.
+
+When talking to Debezium users, we generally receive very good feedback on the range of applications enabled by Debezium and its flexibility: e.g. each connector can be configured and fine-tuned in many ways, depending on your specific requirements. A large number of metrics provide deep insight into the state of running Debezium connectors,
+allowing to safely operate CDC pipelines also in huge installations with thousands of connectors.
+
+All this comes at the cost of a learning curve, though: users new to Debezium need to understand the different options and settings as well as learn about best practices for running Debezium in production.
+We're therefore constantly exploring how the user experience of Debezium can be further improved, allowing people to set up and operate its connectors more easily.
+
+Today it's my great pleasure to introduce you to a proof-of-concept for a potential future *Debezium graphical user interface*.
+The goal for this PoC is to explore how a graphical UI could facilitate the getting started and operational experience of Debezium users.
+
+The scope of the PoC is the set-up flow for configuring and instantiating a Debezium Postgres connector.
+The user is guided through the required configuration steps in a wizard interface,
+starting from mandatory information (e.g. database credentials), over selecting the tables to be captured, up to optional settings like different data mapping options.
+After reviewing the final configuration, the UI will instantiate the connector in Kafka Connect.
+
+You can see a short demo of how this looks like in this video:
+
+++++
+<div class="responsive-video">
+<iframe width="1600" height="900" src="https://www.youtube.com/embed/RZ_3DF7Ndnk" frameborder="0" allowfullscreen></iframe>
```

Comment:
1600 is a very big screen

Reply:
Is this the correct version of the video? There's a small "slip of the tongue" in the first sentences where you are talking about the Debezium team that were building it.

Reply:
> 1600 is a very big screen

This the _maximum_, it's embedded into a responsive div.

> Is this the correct version of the video?

It's the only one I have. It's not perfect, but I thought it's good enough. At which timestamp is this glitch you mean?

Reply:
0:07